essencium-backend/src/main/java/de/frachtwerk/essencium/backend/model/exception/ResourceActions.java

```diff
@@ -0,0 +1,19 @@
+package de.frachtwerk.essencium.backend.model.exception;
+
+public enum ResourceActions {
```

Comment:
If we are using an enum as a source for static strings, do we also want to use a final set of strings? For example, a ResourceAction “IMPORT” or “CHECK” is conceivable. I would either leave out this hard coding or use the HTTP methods as a set.

Reply:
This enum is actually used purely as an util in order to avoid writing manually everywhere the different actions. The exceptions structure itself is not limited to this enum. In the end the action which caused an exception is stored as a String in the respective exception and not as an instance of ResourceActions. I am fine with removing it since there won't be a big difference. We can  input directly as strings the actions where required.

Reply:
What is the current status of this discussion? Do we decide for removing the enum or keeping it?